mobile/lib/utils/pm.dart

```diff
@@ -5,111 +5,6 @@ import 'package:app/themes/theme.dart';
 import 'package:flutter/material.dart';
 import 'package:google_maps_flutter/google_maps_flutter.dart';
 
-List<HealthTip> getHealthTips(
-  double pm2_5,
-  Pollutant _,
-) {
-  final healthTips = <HealthTip>[];
-  if (pm2_5 <= 12.09) {
-    //good
-    healthTips.add(
-      HealthTip(
-        'For everyone',
-        'It\'s a terrific day to go outside and exercise,'
-            ' Reduce the number of car trips you make.',
-        'assets/images/family.png',
```

Comment:
@BenjaminSsempala could we delete these assets since they are nolonger used.

Reply:
@NoahNsimbe  i've removed them